ansible/fhir-stress-test.yml

```diff
@@ -39,142 +23,57 @@
-          - "{{ linux_to_ad_via_ssd_group }}"
-          - "{{ temp_app_pub_group }}"
-        instance_type: "{{ ec2_instance_type }}"
-        image: "{{ ami_id_rhel }}"
-        region: "{{ region }}"
-        vpc_subnet_id: "{{ subnet_id }}" 
-        wait: true
-        # The default of 300s here wasn't cutting it for 8 servers.
-        wait_timeout: 600
-        exact_count: "{{ num_servers }}" 
-        count_tag:
-          Name: bluebutton-stress-server
-          Application: "{{ app_tag }}"
-        instance_tags:
-          Name: bluebutton-stress-server
-          APPLICATION: "{{ app_tag }}"
-          TIER: "{{ tier_tag }}"
-          PURPOSE: bluebutton-stress-server
-          ENVIRONMENT: "{{ env_tag }}"
-          CreatedBy: "{{ whoami.stdout }}"
-      register: ec2_servers
-    
-    - name: Create inventory group 'client'
-      add_host:
-        hostname: "{{ item.private_dns_name }}"
-        groupname: client
-      with_items: "{{ ec2_client.instances }}"
-
-    - name: Create inventory group 'servers'
-      add_host:
-        hostname: "{{ item.private_dns_name }}"
-        groupname: servers
-      with_items: "{{ ec2_servers.instances }}"
-
-    - name: Wait for SSH to come up
-      wait_for:
-        host: "{{ item.private_dns_name }}"
-        port: 22
-        timeout: 320
-        state: started
-      with_flattened:
-        - "{{ ec2_client.instances }}"
-        - "{{ ec2_servers.instances }}"
+- name: Configure Systems for Ansible Pipelining
+  hosts: client:servers
+  vars:
+    ansible_ssh_pipelining: false
+  roles:
+    - rhel_ansible_pipelining
 
 - name: Configure roles common to all servers 
   hosts: client:servers
   become: True
   gather_facts: True
-  environment: "{{proxy_env}}"
+#  environment: "{{proxy_env}}"
   roles:
     - common
 
 - name: Configure roles specific to jmeter client instance
   hosts: client
   become: True
   gather_facts: True
-  environment: "{{proxy_env}}"
+#  environment: "{{proxy_env}}"
   roles:
     - client 
 
 - name: Configure roles specific to jmeter server instance(s)
   hosts: servers
   become: True
   gather_facts: True
-  environment: "{{proxy_env}}"
+#  environment: "{{proxy_env}}"
   roles:
     - server 
 
 - name: Launch JMeter servers
   hosts: servers
   become: True
-  environment: "{{proxy_env}}"
+#  environment: "{{proxy_env}}"
   gather_facts: True
   
   tasks:
     - name: Launch JMeter Server in Background
       # The "JMeter Server" is what the distributed testing nodes run.
-      shell: "( ( nohup {{ remote_jmeter_dir }}/bin/jmeter -s 
+      shell: "nohup {{ remote_jmeter_dir }}/bin/jmeter -s 
         -Djava.rmi.server.hostname={{ ansible_eth0.ipv4.address }}
         -Dserver.rmi.localport={{ jmeter_server_rmi_local_port }} 
         -Dserver_port={{ jmeter_server_port }} 
         -j {{ remote_test_dir }}/log-jmeter.txt
-        &> /dev/null) & )" 
+        &> {{ remote_test_dir }}/log-jmeter-console.txt &" 
```

Comment:
Can you do at least a few minutes of Google research to see if you can figure out why this helped?

Reply:
The parens create a linux subshell. I doesn't seem to be necessary here.